pyproject.toml

```
@@ -29,7 +30,7 @@ packages = [{include = "easyaudit"}]
 
 [tool.poetry.dependencies]
 python = "^3.8"
-django = "^4.2"
+django = ">=4.2,<5.1"
```

Did you mean to include 5.1?

```suggestion
django = ">=4.2,<5.2"
```

---

YES, omgosh.